File: graphql/src/main/java/ru/romanow/protocols/graphql/model/BookResponse.java
package ru.romanow.protocols.graphql.model;

import io.leangen.graphql.annotations.GraphQLQuery;
import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class BookResponse {

    @GraphQLQuery(name = "id")
    private Integer id;
    @GraphQLQuery(name = "name")
    private String name;
    @GraphQLQuery(name = "isn")
    private String isn;
    @GraphQLQuery(name = "price")
    private Integer price;
    @GraphQLQuery(name = "pageCount")
    private Integer pageCount;
    @GraphQLQuery(name = "author")
    private AuthorResponse author;
}

File: graphql/src/main/java/ru/romanow/protocols/graphql/model/BookResponse.kt
package ru.romanow.protocols.graphql.model

data class BookResponse(
    var id: Int? = null,
    var name: String? = null,
    var isn: String? = null,
    var price: Int? = null,
    var pageCount: Int? = null,
    var author: AuthorResponse? = null,
)